### ISSUE TITLE
error in build-kg2.sh

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#954]
```
Traceback (most recent call last):
  File "/home/ubuntu/kg2-code/drugbank_xml_to_kg_json.py", line 389, in <module>
    drugbank_dict = xml_to_drugbank_dict(input_file_name)
  File "/home/ubuntu/kg2-code/drugbank_xml_to_kg_json.py", line 376, in xml_to_drugbank_dict
    drugbank = open(input_file_name)
IsADirectoryError: [Errno 21] Is a directory: '/home/ubuntu/kg2-build/drugbank.xml'
```

**Comment by saramsey** [comment]
From @ericawood it sounds like the fix for this has already been pushed.

**Comment by ericawood** [comment]
This should be fixed with commit 5b563d2413e8049345d7b747b0ad1dfcecc04a9a
